elasticsearch/init/0010-seed-acl

```diff
@@ -23,9 +23,8 @@ index=$( get_security_conf_index )
 if [ -n "${force}" ] ; then
   docs=(roles rolesmapping actiongroups internalusers config)
   for d in $docs; do
-    resp=$(es_util --query=${index}/$d/0 -w %{response_code} | cut -d '}' -f2)
+    if [ check_index_exists "${index}/security/$d" -ne 0 ]; then
```

Comment:
Presumably the doctype changes in future.  Any chance they made it easy for us to migrate in the 6.x version of the security code?

Reply:
I didn't see anything that let us configure the `type` they used